app/services/decisions/income_decision_tree.rb

```diff
@@ -105,16 +125,44 @@ def ended_employment_within_three_months?
       form_object.ended_employment_within_three_months&.yes?
     end
 
+    def appeal_no_changes?
+      crime_app.case.case_type == CaseType::APPEAL_TO_CROWN_COURT.to_s
+    end
+
+    def summary_only?
+      crime_app.case.case_type == CaseType::SUMMARY_ONLY.to_s
+    end
+
     def income_below_threshold?
-      form_object.income_above_threshold.no?
+      crime_app.income.income_above_threshold == 'no'
```

Comment:
Do we need to account for if this answer is nil?

Reply:
Why has this change been made?